src/praisonai-agents/praisonaiagents/runtime/test_resolve.py

```
@@ -0,0 +1,484 @@
+        mock_llm.chat.return_value = "Hello response"
+        
+        wrapper = LLMRuntimeWrapper(
+            llm=mock_llm,
+            model_ref="gpt-4o", 
+            agent_id="test_agent"
+        )
+        
+        assert wrapper.model_ref == "gpt-4o"
+        assert wrapper.provider == "openai"  # inferred from model name
+        assert wrapper.supports_streaming == hasattr(mock_llm, 'stream')
+        assert wrapper.supports_tools == True
```

Ruff (0.15.18)

[error] Avoid equality comparisons to `True`; use `wrapper.supports_tools:` for truth checks

Replace with `wrapper.supports_tools`

(E712)